.circleci/config.yml

```diff
@@ -69,6 +72,13 @@ jobs:
       - image: circleci/node:12
     steps:
       - checkout
+      - run:
```

Comment:
I would prefer leave it as-is as sometimes the PR branch may be conflict with master.

Reply:
So this one is interesting. Here's the problem:

Based on what we previously discussed, we're comparing with the previous master job. Now, if the PR is out of date, then it's not really clear what we're comparing against as the jobs could mean entirely different things if they are not at the same base.

To bring clarity to this situation, I chose to sync with the latest master. This opens us up to the possibility of the job failing if there is merge conflicts, but that's a failure case that'd be remediated anyway. As the author will fix this issue, and the failing job will simply point to the same issue.

But the benefit then, is that it is easy to compare the results and make sense of it..

I'm open to a different direction here

Reply:
Would it be possible to compare with the last common commit between the PR and master?

Reply:
I have thought about this approach before but you will have to run test262 again since the artifact may be out dated if the master has fixed any broken tests.

Another approach may be running on the undocumented `/refs/pull/12345/merge` instead so we don't have to perform git _writing_ operation on CI.

Reply:
> I have thought about this approach before but you will have to run test262 again since the artifact may be out dated if the master has fixed any broken tests.

If it has been fixed in master, but not in the last common commit, than it will still be broken in the PR and it's status shouldn't be shown as changed.

> Another approach may be running on the undocumented /refs/pull/12345/merge instead so we don't have to perform git writing operation on CI.

Better than the current one :+1:

Reply:
So that I'm clear, what's `/refs/pull/12345/merge` and how can I use it?

Reply:
also,

> we don't have to perform git writing operation on CI

it's only a local git writing operation inside CI, afaik, it shouldn't affect the PR or any remote branch.

Reply:
`/refs/pull/PULL_REQUEST_ID/merge` is a undocumented [refspec](https://git-scm.com/book/en/v2/Git-Internals-The-Refspec) on GitHub repositories. You may consider it as a pre-merged version of any pull request. Assume the pull request ID is `12345`, you may fetch and checkout such refs
```
git fetch origin refs/pull/12345/merge
git checkout -qf FETCH_HEAD
```